web/cypress/support/page_objects/landingPage.ts

```diff
@@ -1,14 +1,19 @@
+import { useConfig } from "@/lib/data-hooks/useConfig";
+
+const { Config } = useConfig();
+let landingPageConfig = Config.landingPage;
```

Comment:
@cbrunefearless is the variable necessary?

Reply:
no, just want it to pass first than change :)